mapbox/android-docs
# [docs] add feedback component

We're adding a feedback component to the bottom of each content and example page. It looks like this:

![image](https://user-images.githubusercontent.com/2180540/60911699-c6a0fa00-a251-11e9-9adf-3f6b68d5419b.png)

* [x] update dr-ui `"@mapbox/dr-ui": "^0.18.0",`
* [x] update other dependencies (mr-ui, as needed) `"@mapbox/mr-ui": "^0.7.2",`
* [x] remove `@mapbox/forward-event`, `uuid`, `core-js` as needed
* [x] add feedback module to the markdown page shell (and/or other pages as needed)
  * [x] add the webhook object to constants.json 
  * [x] get username and push to state
* [x] check console for errors on pages with the feedback module
* [x] test to make sure events come through to staging
* [x] run `npm run build` to check for any build-time errors
* [x] Run `npm test` locally to check for broken link warnings

Ref https://github.com/mapbox/documentation/issues/209

### How to test


This branch is on staging

1. `npm install`
1. `npm start`
1. click yes/no on the feedback module
1. open https://app.segment.com/mapbox/sources/mapbox-staging/debugger and search for "sent docs feedback" to see if your events came through (events should come through regardless of ad blocker)
